packages/core/framework/src/config/config.ts

```diff
@@ -164,7 +164,9 @@ export class ConfigManager {
 
     this.#config = {
       projectConfig: normalizedProjectConfig,
-      admin: projectConfig.admin ?? {},
+      admin: projectConfig.admin ?? {
+        path: "/",
```

Comment:
question: Just out of curiousity, why was this added?

Reply:
This is because, the `path` was optional in the duplicated types defined by the `framework` https://github.com/medusajs/medusa/blob/develop/packages/core/framework/src/config/types.ts#L59

Now, if you look at the `admin-bundler` it picks its types from the `@medusajs/types` package and therefore the `path` property is required by it. https://github.com/medusajs/medusa/blob/develop/packages/admin/admin-bundler/src/types.ts#L1

So essentially, framework having its own types could have runtime values mis-match when passing config to the other parts of the stack

Reply:
Maybe I am mistaken, but I believe the default path for admin is `/app`, so this would be a breaking change (that we don't want to do)? I might be wrong though, but might be good to double check the expected behavior here.

Reply:
Yup, you are right!

Reply:
In that case we might need a bit more strict types and here's why.

- Most of the types are written from perspective of what the end-user types.
- However, we go with the assumption that all configs returned from the `medusa-config.ts` file are using the `defineConfig` method and therefore these normalizations have been applied.

In that case, there should be an `InputConfig` type (that we have but not for all nested properties) and then the `OutputConfig` type that reflects the normalizations being applied.